packages/web-components/src/components/cbp-tag/cbp-tag.stories.tsx

```diff
@@ -44,7 +44,7 @@ const Template = ({ label, color, withIcon, context, sx }) => {
       ${context && context != 'light-inverts' ? `context=${context}` : ''}
       ${sx ? `sx=${JSON.stringify(sx)}` : ''}
     >
-      ${Icon ? `<cbp-icon name="${Icon}" sx='{"margin-right":"var(--cbp-space-2x)"}'></cbp-icon>` : ''}
+      ${Icon ? `<cbp-icon name="${Icon}" size="0.75rem"sx='{"margin-right":"var(--cbp-space-1x)"}'></cbp-icon>` : ''}
```

Comment:
Because the icon is the same size as the font size, you shouldn't need this. The default icon size is 1em, which is relative to the surrounding context. Fix the font size (above) and this fixes itself.